android/capacitor/src/main/java/com/getcapacitor/NativePlugin.java

```diff
@@ -1,12 +1,16 @@
 package com.getcapacitor;
 
+import com.getcapacitor.annotation.CapacitorPlugin;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
 /**
  * Base annotation for all Plugins
+ * @deprecated
+ * <p> Use {@link CapacitorPlugin} instead
```

Comment:
Not sure if this was intentional? Does it need to be wrapped in `<p></p>` tags?

```suggestion
 * Use {@link CapacitorPlugin} instead
```

Reply:
It's valid. It doesn't look right but it's how the IDE formatted it correctly, I think. https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format